JavaFXUserInterface/src/main/java/controllers/AssetsController.java

```diff
@@ -306,7 +308,7 @@ public void generateList() {
     /**
      * Stops the timeline
      */
-    private void closeTimeline() {
+    public void closeTimeline() {
         if (rulTimeline != null)
             rulTimeline.stop();
     }
```

Comment:
This method is no longer used, and can probably be deleted.

Reply:
It was deleted